src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -138,8 +139,12 @@ def pluralize(word, nb):
             extras_comments=extras,
             defects_total=pluralize('defect', nb),
             defects='\n'.join(defects),
-            analyzers='\n'.join(analyzers),
         )
+
+        if analyzers:
+            body = COMMENT_RUN_ANALYZERS
+            comment += body.format(analyzers='\n'.join(analyzers))
```

Comment:
Nit: we probably want to keep the newline between the two parts, so you can do
`comment += '\n' + COMMENT_RUN_ANALYZERS.format(...`

(also, don't need the body variable)